.dockerignore

```diff
@@ -1 +1,2 @@
 build
+**/Dockerfile.*
```

Comment:
why it should be ignored?

Reply:
If we don't, then working/modifying the Dockerfile will invalidated the build cache, since we are doing `COPY . .`

In fairness, this is a micro-optimization but it annoyed it and made my workflow slower. :)